run_flask.py

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/python3
+from src.dashboard.webserver.server import app
+
+if __name__ == "__main__":
+    app.run(debug=True, port=5000, host="0.0.0.0")
```

Comment:
Two changes:

- `debug=False`
- `use_reloader=False`

Reply:
You already mentioned debugging, but what does the reloader do?

Reply:
So the reloader will reload the flask server when the code changes, and this will consume quite a bit of memory.

See here: https://flask.palletsprojects.com/en/0.12.x/errorhandling/